internal/app/router_authorization_test.go

```
@@ -72,6 +88,34 @@ func TestRegisterRoutes_楽曲追加削除はEDITORを拒否する(t *testing.T)
 }
 
+func TestRegisterRoutes_公開GETはread最適化認証を使い書き込みはstrict認証を使う(t *testing.T) {
+	// Given
+	e := echo.New()
+	e.HTTPErrorHandler = appmiddleware.CustomHTTPErrorHandler
+	strictAuth := &countingAuthenticator{}
+	readOptimizedAuth := &countingAuthenticator{}
+	registerRoutes(e, newAuthorizationTestHandlers(), strictAuth, readOptimizedAuth, nil, config.Config{})
+
+	// When
+	getReq := httptest.NewRequest(http.MethodGet, "/internal/songs", nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)